src/doomdef.h

```diff
@@ -30,7 +30,9 @@
 
 #include <stdio.h>
 #include <string.h>
-
+#ifdef __FreeBSD__
+#include <ctype.h>
+#endif
```

Comment:
I think `ctype.h` can be included unconditionally.

BTW, are those functions used so frequently? I mean how many .cpp files should be changed?

Reply:
They're used in at least two files, if not more. I don't know how many files they're actually used in, but with the localization stuff not even being finished yet, and those functions often being regarded as "convenience functions" - it would not surprise me if they pop up more, in the future. I honestly don't know of a better way to handle this situation - the only CI service that handles FreeBSD is Cirrus-CI, so if someone does another change again somewhere else, unless that's added to this repo it will just break again anyhow.

As for including it unconditionally - true that might be viable, but I chose not to do it because it wasn't necessary. Only one platform needs it right now, that I know of.

Reply:
It will be included on all platforms anyway, just indirectly.

Reply:
Which makes me even more leery of including it unconditionally. But that's just me. Until either you or Graf tell me directively to remove the #ifdef guard, I will leave it there, because like I said, I am not sure what's really the best way to handle it. I don't feel great about including a file if it's already included, and this is really only needed for FreeBSD right now.